src/styles/blocks/catalog.scss

```
@@ -17,9 +17,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-
+    
+    transition: transform 0.3s, color $trans-duration;
```

Use variable you've created:
```suggestion
    transition: transform $trans-duration, color $trans-duration;
```